contxt/models/facilities.py

```diff
@@ -1,48 +1,662 @@
-from dataclasses import dataclass
-from typing import ClassVar
-
-from . import ApiField, ApiObject, Parsers
-from .contxt import Organization
-
-
-@dataclass
-class Facility(ApiObject):
-    _api_fields: ClassVar = (
-        ApiField("id", data_type=int),
-        ApiField("name"),
-        ApiField("address1"),
-        ApiField("address2"),
-        ApiField("city"),
-        ApiField("state"),
-        ApiField("zip"),
-        ApiField("timezone"),
-        ApiField("geometry_id"),
-        ApiField("asset_id"),
-        ApiField("tags"),
-        ApiField("organization_id"),
-        ApiField("Organization", attr_key="organization", data_type=Organization),
-        ApiField(
-            "Info", attr_key="info", data_type=lambda o: {k: Parsers.unknown(v) for k, v in o.items()}
-        ),
-        ApiField("created_at", data_type=Parsers.datetime),
-        ApiField("weather_location_id"),
-        ApiField("slug"),
+from sgqlc.types import Arg, Enum, Field, Input, Type, list_of, non_null
```

Comment:
was this generated or manually written?

Reply:
Taken from the codegen output but I was having some trouble getting it working here, so manually written with a template